io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
```

Comment:
Given how appimages are traditionally used, you probably want to use `--filesystem=~/Applications:create` here, or move it completely to `$XDG_DATA_HOME` (applications have write access here by default). Probably the `--filesystem` makes more sense to make it easier for users to find and execute download applications outside the sandbox.